test/Common/DurableHttpTests.cs

```diff
@@ -1509,6 +1547,21 @@ private static HttpMessageHandler MockSynchronousHttpMessageHandlerWithTimeoutEx
             return handlerMock.Object;
         }
 
+        private static HttpMessageHandler MockSynchronousHttpMessageHandlerWithHttpRequestException()
+        {
+            HttpResponseMessage httpResponseMessage = CreateTestHttpResponseMessage(HttpStatusCode.OK);
```

Comment:
Is there a reason why we reply status-code "OK" even though the message failed? I realize this is just a test, but it did throw me off :)

Reply:
There's no particular reason, it's just there as a dummy value to create a `HttpResponseMessage`. The important part of this test is on line 1554 where the content stores an `HttpRequestException`. I do see how it can be confusing though so I can change it to `NotFound`.